### ISSUE TITLE
Background text contrast should be increased in Date and time pickers

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14400]
I got that piece of feedback from an a11y review. Are we following material guidelines here?

**Comment by Hixie** [comment]
cc @darrenaustin

**Comment by Hixie** [comment]
(If I understand correctly, we're working on revamping these? cc @HansMuller)

**Comment by darrenaustin** [comment]
We are definitely revamping the Date Picker as we are implementing the latest Material design and adding in the range picker. I don't know that we have plans for the Time Picker, but I can take a look at that after we finish the Date Picker.

Do we have specifics about the complaints? Exactly what text are they referring to?
